packages/sdk/generated/auth/client.ts

```diff
@@ -605,6 +620,20 @@ export class AuthClient {
     return this.listCredentials()
   }
 
+  async listOrgSettings(): Promise<T.ListOrgSettingsResponse> {
```

Comment:
I think org settings are suppose to be dashboard only endpoints?

Reply:
they are, but this is done through the sdk generation, is there a flag to put somewhere for it not to be generated?

Furthermore, aren't we planning to use the SDK to refactor the dashboard? Should we publish 2 SDKs?

Reply:
if you don't add to the auth public endpoints, generator won't include the endpoint. yeah we will want to have a separate lib for other endpoints.

Reply:
Are we already segregating Auth Public Endpoints and Auth Dashboard Public Endpoints?
If not, I don't think I can remove it from the auth public endpoints?

Are we talking about this one: https://github.com/dfnsco/dfns/blob/m/services/auth/libs/enclave-auth-library/src/verification/hostConfig/publicApiHostConfig.ts#L18

or this one: https://github.com/dfnsco/dfns/blob/m/services/auth-management/libs/auth-management-domain/src/public/endpoints/index.ts#L101 ?